README.md

```diff
@@ -39,10 +39,56 @@ make plugin
 kubectl cm
 oc cm
 ```
+
 ## Disclaimer
 
 This CLI (and plugin) is still in development, but aims to expose OCM/ACM's functional through a useful and lightweight CLI and kubectl/oc CLI plugin.  Some features may not be present, fully implemented, and it might be buggy!  
 
+## Getting Started
+
+### Setting up a ClusterPoolHost
+
+In order to work with clusters, you need set up `cm` with your hub cluster(s) - `cm` refers to these hubs as "clusterpoolhost"(s) or "cph"(s) for short!  
+
+To set up your first ClusterPoolHost:
+1. `oc login` to your ClusterPoolHost running [Red Hat Advanced Cluster Management](https://access.redhat.com/products/red-hat-advanced-cluster-management-for-kubernetes), [Multicluster Engine for Kubernetes](https://github.com/open-cluster-management/backplane-operator), or [Open Cluster Management](http://github.com/open-cluster-management-io).  **Your user must be able to create ServiceAccounts in the target namespace, given that `create cph` creates a ServiceAccount.  Also ensure that ServiceAccounts in that namespace have the relevant access such as create/delete clusterclaims, pools, etc.   
```

Comment:
It might be helpful to include the full list of resources that require permission, as well as the API group and version.

Reply:
I'll add as much detail as I can but alas its... a bit lossy.